src/test/java/io/spine/helloworld/server/hello/HelloContextTest.java

```diff
@@ -11,13 +12,11 @@
 import static io.spine.testing.TestValues.randomString;
 
 @DisplayName("Hello Context should")
```

Comment:
Shouldn't we backtick the name?

Reply:
We don't use ticks here because none of the words are code.